web/rest/brand/app/config/api/resources.yml

```diff
@@ -50,3 +50,24 @@ Model\RatingPlanPrices:
             description: 'Rating plan group Id'
   attributes:
     pagination_client_enabled: true
+
+Model\ActiveCalls:
+  itemOperations:
+    get_my_active_calls:
+      access_control: '"ROLE_BRAND_ADMIN" in roles'
+      method: 'GET'
+      path: '/my/active_calls'
```

Comment:
Out of curiosity, will be _my_ the _namespace_ for all custom endpoints?  Or it's also possible to use _Provider_?

Reply:
There is no technical limitation about namespaces